draft-ietf-rtcweb-jsep.xml

```diff
@@ -3623,20 +3623,55 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <t>Construct a table mapping MID to RtpReceiver for each
           RtpReceiver configured to receive from this transport.</t>
 
-          <t>Construct a table mapping SSRC to RtpReceiver for each
-          RtpReceiver configured to receive from this transport and for
-          each SSRC that RtpReceiver is configured to receive. Some of
-          the SSRCs may be presesnt in the m= section corresponding to
-          that RtpReceiver in the remote description.</t>
+          <t>Construct a table mapping incoming SSRC to RtpReceiver
+          for each RtpReceiver configured to receive from this
+          transport and for each SSRC that RtpReceiver is configured
+          to receive. Some of the SSRCs may be present in the m=
+          section corresponding to that RtpReceiver in the remote
+          description.</t>
+
+          <t>Construct a table mapping outgoing SSRC to RtpSender for
+          each RtpSender configured to transmit from this transport
+          and for each SSRC that RtpSender is configured to use when
+          sending.</t>
 
           <t>Construct a table mapping payload type to RtpReceiver for
           each RtpReceiver configured to receive from this transport
           and for each payload type that RtpReceiver is configured to
           receive. The payload types of a given RtpReceiver are found
           in the m= section corresponding to that RtpReceiver in the
-          local description. If any payload type could map to more than
-          one RtpReceiver, map to the RtpReceiver whose m= section
-          appears earliest in the local description.</t>
+          local description. If any payload type could map to more
+          than one RtpReceiver, map to the RtpReceiver whose m=
+          section appears earliest in the local description.</t>
+        </list>
+      </t>
+
+      <t>For each RTCP packet received, the following steps MUST be
+      followed:</t>
+
+      <t>
+        <list>
+          <t>For each SDES item in the RTCP packet that contains a
+          MID that is in the table mapping MID to RtpReceiver, update
+          the incoming SSRC mapping table to include an entry the SSRC
+          for that SDES item to the RtpReceiver associated with the
+          MID.</t>
+
+          <t>For each Receiver Report in the RTCP packet, if the SSRC
+          for the Receiver Report is found in the outgoing SSRC table,
+          deliver a copy of the RTCP packet to the RtpSender found in
+          the outgoing SSRC table.</t>
+
+          <t>For each Sender Report in the RTCP packet, if the SSRC
```

Comment:
I think this also needs to go to the RtpSender as well, since a SR contains all RR information as well.